index.html

```diff
@@ -1,32 +1,111 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>My Great Page!</title>
-    <!-- LESS -->
-    <link rel="stylesheet/less" type="text/css" href="src/css/main.less" />
-    <script>
-      less = {
-        env: "development",
-        logLevel: 2,
-        async: false,
-        fileAsync: false,
-        poll: 1000,
-        functions: {},
-        dumpLineNumbers: "comments",
-        relativeUrls: true
-      };
-    </script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/less.js/3.9.0/less.min.js"></script>
-    <script>
-      less.watch();
-    </script>
-  </head>
-  <body>
-    <div>
-      <h1>Create Great Ideas here!</h1>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="theme-color" content="#001f3f">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="apple-touch-icon" sizes="180x180" href="src/assets/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="src/assets/favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="src/assets/favicon/favicon-16x16.png">
+  <link rel="manifest" href="src/manifest.json">
+  <title>Library</title>
+  <!-- LESS -->
+  <link rel="stylesheet/less" type="text/css" href="src/css/main.less" />
+  <script>
+    less = {
+      env: "development",
+      logLevel: 2,
+      async: false,
+      fileAsync: false,
+      poll: 1000,
+      functions: {},
+      dumpLineNumbers: "comments",
+      relativeUrls: true
+    };
+  </script>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/less.js/3.9.0/less.min.js"></script>
+  <script>
+    less.watch();
+  </script>
+</head>
+
+<body>
+  <header>
+    <img src="/src/assets/icons/research.png" alt="logo">
```

Comment:
[Optional]

Let's add "." to the path or remove the first "/", currently, the images are not showing. See https://raw.githack.com/fabricio-garcia/library/6a210fa87583b62cf1603a2919de1d33b8e11a2d/index.html

```suggestion
    <img src="./src/assets/icons/research.png" alt="logo">
```

Let's do this for all images added to the app.

Reply:
**SOLVED!**